zeppelin-web-angular/src/index.html

```diff
@@ -17,7 +17,6 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" href="./assets/images/zeppelin.png" type="image/x-icon">
   <title>Zeppelin</title>
-  <base href="/">
```

Comment:
You take this out in the new UI, which I understand. Where is the place in the old UI?

Reply:
Both the old and new UIs are served from their respective sub-paths (depending on the configuration, either `/`, `/new`, or `/classic`), and as I understand it, the trailing part of the URL after `#/...` is treated as an anchor and ignored.
Therefore, without the base tag, static resources should be correctly fetched from the appropriate web app serving path, since the `src` attributes for those resources are set using relative paths.
I verified this in the browser, and everything worked as expected.

Given this, I believe neither app requires a base tag. In fact the old UI didn't have one to begin with.

I understand your question as asking where the base tag is in the old UI, so I've responded accordingly.
However, if I misunderstood your point, please feel free to clarify.